common/logger/src/libra_logger.rs

```diff
@@ -132,6 +134,11 @@ impl LibraLoggerBuilder {
         self
     }
 
+    pub fn remote_level(&mut self, level: Level) -> &mut Self {
+        self.central_level = level;
+        self
```

Comment:
Could we use the same terms for the `level` / `central_level` with the `local_level` / `remote_level`?

Reply:
oops thanks for catching this